<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1311</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWebEngineView" name="TLEVIew" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>761</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>761</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Iridium">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>361</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Update Iridium TLE from Network</string>
   </property>
  </widget>
  <widget class="QPushButton" name="IridiumNext">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>90</y>
     <width>361</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Update Iridium NEXT TLE from Network</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>130</y>
     <width>491</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="UpdateTLE">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>90</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>90</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>591</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 48pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>TLE Pre-Update Tool</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>35</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>By sunguiyu96@gmail.com</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
